css/style.css

```diff
@@ -42,18 +42,72 @@ Developer @GA in NYC
 /* BASE STYLES
 /*******************************************/
 
-body {
+* {
```

Comment:
Its better to apply these global typography rules to the `html` element rather than using the wildcard (`*`).